config/routes.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  namespace :api do
+    resources :greetings, only: [:index] do
+      collection do
+        get 'random_greeting'
+      end
+    end
+  end
+  # Defines the root path route ("/")
+  # root "articles#index"
```

Comment:
- [ ] Great job working on this milestone. Kindly remove all commented lines of code from your codebase. This makes it look clean and professional